src/position/orca-position.ts

```diff
@@ -75,18 +75,18 @@ export class OrcaPosition {
 
     const txBuilder = new TransactionBuilder(ctx.provider);
 
-    const { address: tokenOwnerAccountA, ...tokenOwnerAccountAIx } = await resolveOrCreateATA(
+    const [ataA, ataB] = await resolveOrCreateATAs(
```

Comment:
if tokenMintA is NATIVE_MINT, i think resolveOrCreateATAs would place it at the back of the instruction queue (becoming ataB)